src/DataViews/SamplingReportDataView.cpp

```diff
@@ -340,6 +339,34 @@ void SamplingReportDataView::SetStackEventsCount(uint32_t stack_events_count) {
   stack_events_count_ = stack_events_count;
 }
 
+std::string SamplingReportDataView::GetToolTip(int row, int column) {
+  if (column != kColumnInclusive && column != kColumnExclusive) {
+    return "";
+  }
+  const SampledFunction& function = GetSampledFunction(row);
+  uint32_t raw_count = {};
+  float percentage = {};
+  if (column == kColumnInclusive) {
+    raw_count = function.inclusive;
+    percentage = function.inclusive_percent;
+  } else {
+    raw_count = function.exclusive;
+    percentage = function.exclusive_percent;
+  }
+
+  orbit_statistics::BinomialConfidenceInterval interval =
+      app_->GetConfidenceIntervalEstimator().Estimate(percentage / 100.0f, stack_events_count_);
+
+  return absl::StrFormat(
+      "The function %s\n"
+      "has been encountered %u times in a total of %u stack samples.\n"
```

Comment:
I would say "has been" -> "was" as the capture is over but of course I'm not a native...

Reply:
This is about whether there is a continuation in the present or not.
has been encountered -> so far, more could be encountered
was encountered -> the number is fixed